+ zh-HK main page, replacing zh-CN copy

JeromeMartinez opened the pull request.

XhmikosR commented:
And the menu?

JeromeMartinez commented:
The menu is modified in _.html

XhmikosR commented:
No, I mean, should you remove zh-HK from the menu too? Or this is just a duplicate file?

JeromeMartinez commented:
I don't see the issue.
Previous zh-HK/_.html was duplicated content from zh-HK/_.html, replaced with this commit by real zh-HK. Nothing removed, just replaced.

JeromeMartinez commented on README.md:
```diff
@@ -1,2 +1,68 @@
+## Dependencies
+
+* Apache web server >= 2.2 (should also work on nginx and lighttpd)
+    * mod rewrite is recommended
```
mod\_rewrite?

JeromeMartinez commented on README.md:
```diff
@@ -1,2 +1,68 @@
+
+## Dependencies
+
+* Apache web server >= 2.2 (should also work on nginx and lighttpd)
```
>= 2.2 mandatory or just not tested?
Suggest "A web server (tested on Apache 2.2)"

JeromeMartinez commented on README.md:
```diff
@@ -1,2 +1,68 @@
+
+* Apache web server >= 2.2 (should also work on nginx and lighttpd)
+    * mod rewrite is recommended
+* Php >= 7.0
```
>= 7.0 mandatory?
Suggest "PHP (tested on version 7.0 and 7.1)"

JeromeMartinez commented on README.md:
```diff
@@ -1,2 +1,68 @@
+* Apache web server >= 2.2 (should also work on nginx and lighttpd)
+    * mod rewrite is recommended
+* Php >= 7.0
+    * Mandatory packages for debian-like distributions : libapache2-mod-php7.0, php7.0-cli, php7.0-intl, php7.0-curl, php7.0-opcache
```
"Debian" (uppercase as RedHat)

JeromeMartinez replied:
directly the command ("apt ..." and "yum ...") as it is done with "git clone ...".
